src/test/java/teammates/storage/search/StudentSearchTest.java

```diff
@@ -10,7 +10,7 @@
 import teammates.common.datatransfer.attributes.CourseAttributes;
```

Comment:
I think its better if we test that the exception messages are correctly returned

Reply:
You're right, I'll add on to the tests in another PR which aims to improve testing for the new search service.